generated/nodejs/google-cloud-node/packages/errorreporting/package.json

```diff
@@ -3,19 +3,25 @@
   "name": "@google-cloud/errorreporting",
   "version": "0.7.1",
   "author": "Google Inc.",
-  "description": "Stackdriver Clouderrorreporting library for Node.js",
+  "description": "Errorreporting client for Node.js",
   "main": "src/index.js",
   "files": [
     "src"
   ],
   "dependencies": {
-    "google-proto-files": "^0.7.0",
+    "google-proto-files": "^0.8.3",
     "google-gax": "^0.7.0",
```

Comment:
Use google-gax version `0.10.5` (or `0.10.6`?  @landrito -- are you gonna republish a newer version?)